src/Sets/Star.jl

```diff
@@ -4,7 +4,7 @@ export Star,
        predicate
 
 """
-    Star{N, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}, PT<:AbstractPolyhedron{N}} <: AbstractStar{N}
+    Star{N, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}, PT<:AbstractPolyhedron{N}}
```

Comment:
I am not sure about this line. If you look at the output in the doctest, you get this:
```suggestion
    Star{N, PT<:AbstractPolyhedron{N}, NM, MAT<:AbstractMatrix{NM}, VN<:AbstractVector{NM}}
```
Maybe just drop the parameters altogether?

Reply:
true, i fixed it to match the functions signature